### DISCUSSION TITLE
Non-Synced Parameter Support?

### DISCUSSION THREAD

**Discussion by Entorix** [vrchat-community/osc#94] [Q&A]
Would it be possible to add support for non-synced parameters to OSC input/output? 
I can think of a few instances where they wouldn't need to be synced and could trigger other synced parameters. 
In my specific case, I need to be able to control something that is local only using a float value, but I don't have enough bits left on the avatar to facilitate this.

This is more a question aimed at the Devs, but input is welcome!

**Comment by Janni9009** [comment]
I would love to see this as well since it would allow us to really compact down our synced params with some binary logic for our fancy new custom menus and some parameter drivers, like combining a whole bunch of bool toggles into various outputs on the same int or even compressing multiple ints down into one that's not even near it's maximum capacity normally!
